stanford/roles/cluster/defaults/main.yml

```diff
@@ -178,6 +184,15 @@ CLUSTER_SECURITY_GROUP_RULES:
       from_port: 25672
       to_port: 25672
       cidr_ip: "{{ DEPLOYMENT_CIDR }}"
+  sandbox:
+    - proto: tcp
+      from_port: 22
+      to_port: 22
+      cidr_ip: '0.0.0.0/0'
+    - proto: tcp
+      from_port: 80
+      to_port: 80
+      cidr_ip: '0.0.0.0/0'
```

Comment:
I know we'll need to open more ports later (Studio comes to mind),
but just starting with the LMS for now.

Reply:
Actually, I think we're good here.
Studio comes in via nginx at `studio.whatever`.